### PR TITLE
pijul: 0.10.0 -> 0.11.0

### PR DESCRIPTION
**Pull request by RohanHart**
###### Motivation for this change
update to latest

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/version-management/pijul/default.nix [hunk 1]
```diff
@@ -4,16 +4,16 @@ with rustPlatform;
...
+    url = "https://pijul.org/releases/pijul-0.11.0.tar.gz";
+    sha256 = "e60793ab124e9054c1d5509698acbae507ebb2fab5364d964067bc9ae8b6b5e5";
   };
 
-  cargoPatches = [
-    ./libpijul.patch
-  ];
+  # cargoPatches = [
+  #   ./libpijul.patch
+  # ];
```
Comment (vbgl) [inline pkgs/applications/version-management/pijul/default.nix]: If the patch is no longer needed, these lines and the file should be removed.

File: pkgs/applications/version-management/pijul/default.nix [hunk 2]
```diff
@@ -23,13 +23,13 @@ buildRustPackage rec {
     $out/bin/pijul generate-completions --zsh > $out/share/zsh/site-functions/_pijul
     $out/bin/pijul generate-completions --fish > $out/share/fish/vendor_completions.d/pijul.fish
   '';
-
```
Comment (vbgl) [inline pkgs/applications/version-management/pijul/default.nix]: Please don’t add unneeded white space.

**Review by vbgl — CHANGES_REQUESTED** [review]

File: pkgs/applications/version-management/pijul/default.nix (hunk 1 above)
Comment (RohanHart) [reply pkgs/applications/version-management/pijul/default.nix]: removed

File: pkgs/applications/version-management/pijul/default.nix (hunk 2 above)
Comment (RohanHart) [reply pkgs/applications/version-management/pijul/default.nix]: removed

File: pkgs/applications/version-management/pijul/default.nix [hunk 3]
```diff
@@ -4,17 +4,13 @@ with rustPlatform;
 
 buildRustPackage rec {
   name = "pijul-${version}";
-  version = "0.10.0";
+  version = "0.11.0";
 
   src = fetchurl {
     url = "https://pijul.org/releases/${name}.tar.gz";
-    sha256 = "1lkipcp83rfsj9yqddvb46dmqdf2ch9njwvjv8f3g91rmfjcngys";
+    sha256 = "e60793ab124e9054c1d5509698acbae507ebb2fab5364d964067bc9ae8b6b5e5";
```
Comment (RohanHart) [inline pkgs/applications/version-management/pijul/default.nix]: Note that this is the sha256 published at https://pijul.org/downloads/
Should I convert this to base32?

Comment (worldofpeace) [reply pkgs/applications/version-management/pijul/default.nix]: @RohanHart 
If you want.

You can use ` nix-hash --to-base32 --type sha256`.

**Comment by dywedir** [comment]
@GrahamcOfBorg build pijul

**Review by dywedir — APPROVED** [review]
Thanks!

**Review by vbgl — APPROVED** [review]

**Comment by P-E-Meunier** [comment]
Great, but why not use Carnix instead? I wrote it especially for this.

**Comment by RohanHart** [comment]
> Great, but why not use Carnix instead? I wrote it especially for this.

It was the path of least resistance.

**Pull request merged**